ui/components/component-library/avatar-account/avatar-account.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import Jazzicon from '../../ui/jazzicon/jazzicon.component';
+import BlockieIdenticon from '../../ui/identicon/blockieIdenticon/blockieIdenticon.component';
+import { BaseAvatar } from '../base-avatar';
+
+import { SIZES } from '../../../helpers/constants/design-system';
+
+export const diameters = {
```

Comment:
This doesn't seem to be used anywhere else? So does it need to be exported? And would we adopt the uppercase convention for a const?